flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -1503,6 +1503,12 @@ bool OmpAttributeVisitor::Pre(const parser::OpenMPBlockConstruct &x) {
   const auto &beginBlockDir{std::get<parser::OmpBeginBlockDirective>(x.t)};
   const auto &beginDir{std::get<parser::OmpBlockDirective>(beginBlockDir.t)};
   switch (beginDir.v) {
+  case llvm::omp::Directive::OMPD_masked_taskloop_simd:
```

Comment:
Remove TASKLOOP-related enums from here and `Post(const parser::OpenMPBlockConstruct &)` and instead introduce them in the `Pre` method for `OpenMPLoopConstruct`.

Reply:
Done